config/orderValidator.php

```diff
@@ -0,0 +1,1957 @@
+<?php
+
+declare(strict_types=1);
+
+use MyParcelNL\Pdk\Base\Service\CountryService;
+use MyParcelNL\Pdk\Carrier\Model\CarrierOptions;
+use MyParcelNL\Pdk\Shipment\Model\DeliveryOptions;
+
+return [
+    'platform' => [
+        [
+            'name'    => 'myparcel',
+            'human'   => 'MyParcel',
+            'base_cc' => CountryService::CC_NL,
+            'carrier' => [
+                [
+                    'id'           => CarrierOptions::CARRIER_POSTNL_ID,
+                    'name'         => CarrierOptions::CARRIER_POSTNL_NAME,
+                    'human'        => 'PostNL',
+                    'requirements' => [
+                        'largeFormat' => [
+                            true => [
+                                'schema' => [
+                                    'type'                 => 'array',
+                                    'additionalProperties' => true,
+                                    'properties'           => [
+                                        'physicalProperties' => [
+                                            'type'                 => 'array',
+                                            'additionalProperties' => true,
+                                            'properties'           => [
+                                                'weight' => [
+                                                    'type'    => 'integer',
+                                                    'minimum' => 0,
+                                                    'maximum' => 30000,
+                                                ],
+                                            ],
+                                        ],
+                                    ],
+                                ],
+                            ],
+                        ],
+                        'ageCheck'    => [
+                            true => [
```

Comment:
true als key? dit is best wel "raar en eng" en gaat ook fout wanneer er waarden komen die niet als key kunnen. ik weet dat we nog vastzitten aan een oude json schema draft, maar als je dit toch op json schema wil laten lijken, implementeer dan zelf die if/then/else logica https://json-schema.org/understanding-json-schema/reference/conditionals.html#id6